snap/snapcraft.yaml

```diff
@@ -0,0 +1,90 @@
+name: notes
+# Note on version: $UPSTREAM_VERSION~.git$GIT_COMMIT_ID
+summary: Note taking application, write down your thoughts.
+description: |
+  Notes is a tool for the things you need to write down off your
+  brain. It is your place of expressing yourself. Notes is an open
+  source and cross-platform project, and you're welcome to be part
+  of it and contribute. We hope you will love it as much as we do.
+version: latest
+version-script: GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD);
```

Comment:
Note that `version-script` is [deprecated](https://snapcraft.io/docs/deprecation-notices/dn10), so while you're updating the `snapcraft.yml` file you may want to use the new method for setting the version.

For Tiled I ended up with:

https://github.com/bjorn/tiled/blob/9cc4bcd6322cd2d7b73a2de1f8a594749bd29837/snap/snapcraft.yaml#L37-L39

Reply:
I wasn t aware of that, your repo is one of my references ;) I'll update that.

Reply:
Hehe, though in general you have to be a bit careful using it as reference. Most of the snap packaging stuff is just guesswork or copy/paste... Please let me know if you find anything that could be improved. :-)